viewing_party/play_test.py

```diff
@@ -0,0 +1,253 @@
+# # def get_available_recs(user_data):
```

Comment:
Commented old code like this is good to remove before committing and/or turning in the assignment in the future.